crates/providers/providers-local/src/chain_provider.rs

```diff
@@ -136,7 +136,7 @@ impl InMemoryChainProviderInner {
                     blob_gas_used: header.blob_gas_used,
                     excess_blob_gas: header.excess_blob_gas,
                     parent_beacon_block_root: header.parent_beacon_block_root,
-                    requests_hash: header.requests_hash(),
+                    requests_hash: header.requests_hash,
                 },
```

Comment:
Should this just be an `Into::into()` too? Does reth Chain header implement `Into<alloy_consensus::Header>`